mixer/pkg/runtime2/handler/env.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2018 Istio Authors
```

Comment:
Can this file be somehow shown as branched from istio/mixer/pkg/runtime/env.go ?

Reply:
I tried it with the config code, but it didn't hold. I suspect it has to do with branching & editing in the same commit. (Or GoLand doesn't use git to track moves, and simply adds the files).
I'll try to do it as a separate update to this if possible and update the PR.

Reply:
Hmm, simpler reason (and more complicated work around) than I thought:
https://stackoverflow.com/a/44036771